docs/API_action_commands.md

```diff
@@ -0,0 +1,31 @@
+# Action API supported commands
+
+## Supported actions
+These actions are currently supported using the [Action API](API.md#ActionAPI)
+
+None at this time.
+
+----
+## Actions under development
+These actions are under active development
+
+* deploy site  
+Triggers the initial deployment of a site, using the staged design and secrets
+
+* update site  
```

Comment:
We would need to think carefully on what exactly upgrade would do that update wouldn't.  If its a matter of _hold onto your seat you're about to receive a huge change, potentially an OpenStack upgrade, and we'd like you to snapshot the world_ sort of difference, perhaps that is best handled by preceeding the call with an action to archive, snapshot, whatever it may be.  One of my goals is everything using the same paths so we have well tested/used paths for minor and major changes.